app/controllers/concerns/load_settings.rb

```diff
@@ -18,6 +18,12 @@ def build_settings_defaults
         website_logo_url: '',
         website_url: ''
       })
+      SettingsObject.create(title: 'styles', values: {
+        css: ''
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.